src/main/resources/jsonSchemas/antoraComponentSchema.json

```diff
@@ -20,13 +19,22 @@
       "type": [
         "string",
         "number",
+        "boolean",
+        "object",
         "null"
       ],
       "description": "The version key is the version coordinate in the resource ID, version segment in the page URL, and, when display_version isn’t set, the version name displayed in the UI. It’s also used to sort multiple versions of the same component. Use '~' (without quotes) for a versionless component. Examples: '1', 'v2.0-beta', 'rawhide'",
       "examples": [
         "1",
         "v2.0-beta",
-        "rawhide"
+        "rawhide",
+        true,
+        {
+          "version": {
+            "v(?<version>+({0..9}).+({0..9})).x": "$<version>",
+            "feature/(*)/*": "$1"
+          }
+        }
```

Comment:
That's the example given at https://docs.antora.org/antora/latest/component-version-key/#refname-projection

Reply:
Thanks, look good. Still haven't figured out how to show these in the IDE, though.